com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs

```diff
@@ -18,7 +19,7 @@ class VFXGraphPreprocessor : AssetPostprocessor
     {
         static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
         {
-            List<string> assetToReimport = null;
+            List<string> assetToReimport = new List<string>();
```

Comment:
OnPostprocessAllAssets is called for every import, we should always prevent any unecessary allocation in this callback. Even if the impact is minimal, the memory profiling will be noisy for anyone trying to debug an asset import.
We should definitely let the lazy assetToReimport invocation here.

Reply:
Ok, reverted